### PR TITLE
switch to nsp

### PR DESCRIPTION
**Pull request by flenter**
bump version to 2.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 # Node security shrinkwrap audit
 
-This step will send the shrinkwrap file to nodesecurity.io. If any vulnerable modules are found, than this step will fail.
+This step will install/run the [nsp](https://github.com/nodesecurity/nsp) 
+command, which will be used to check your npm-shrinkwrap.json and node\_modules
```
Comment (hatchan) [inline README.md]: `node\_modules` -> `node_modules`

File: README.md [hunk 2]
```diff
@@ -16,22 +19,11 @@ build:
...
-
 # What's new
 
-- Initial release.
-
-# Options
+- switch to nsp command/module
 
-- `shrinkwrap-path` (optional) The path to the shrinkwrap file. Defaults to `./npm-shrinkwrap.json`.
+- Initial release.
```
Comment (hatchan) [inline README.md]: This is not the initial release anymore

File: wercker-step.yml [hunk 3]
```diff
@@ -1,9 +1,8 @@
 name: shrinkwrap-security-audit
-version: 1.0.0
-description: A step to check the npm-shrinkwrap.json file for vulnerable modules using nodesecurity.io
+version: 2.0.0
+description: A step that runs the nsp command
 keywords:
   - security
   - nodejs
-  - shrinkwrap
```
Comment (hatchan) [inline wercker-step.yml]: why is this removed? it still is auditing the npm-shrinkwrap file, right?

File: run.sh [hunk 4]
```diff
@@ -1,68 +1,2 @@
...
-    if is_file "$shrinkwrap_path"; then 
-        security_audit "$step_path" "$shrinkwrap_path"
-    else 
-        fail "Shrinkwrap file was not found"
-    fi
-}
-
-main
\ No newline at end of file
+sudo npm install nsp -g
```
Comment (hatchan) [inline run.sh]: Why do the installation at runtime, and not at "buildtime"?

File: README.md (hunk 1 above)
Comment (flenter) [inline README.md]: I did that on purpose, seems like github markdown does not require it. Removing it

File: run.sh [hunk 5]
```diff
@@ -1,68 +1,2 @@
...
+nsp check
```
Comment (hatchan) [inline run.sh]: When I said, we should use the official module, I meant use the library in our javascript code.

Comment (hatchan) [inline run.sh]: Now we're also not able to specify a different shrinkwrap path anymore :-/

Comment (flenter) [inline run.sh]: I think checking the installed makes the step more valuable. But it indeed changes the behaviour of the step

File: run.sh (hunk 4 above)
Comment (flenter) [inline run.sh]: The step tool is usually installed globally, so it required more testing. Since I wanted to quickly fix the step (and builds using it) I implemented this quick solution, which is (nearly) identical to some of our other steps (like the jshint step)

**Comment by flenter** [comment]
I've taken a different approach, closing this pr in favor of: https://github.com/wercker/step-shrinkwrap-security-audit/pull/2
